style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: rgb(225, 245, 245);
+}
```

Comment:
You have a great implementation in your HTML and CSS, so it will be great if you present your Pull Request (PR) in a [very professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). To achieve this remember that it's very important to have in your Pull Request a description with a short summary explaining introduced changes in more detail.

- [ ] Kindly add a nice description for your project 😄 .